### PR TITLE
Added JS tip calc

### LINKED ISSUE #156: Tip Calculator
**Project title**
Tip Calculator

**Project Description**
This script consists of user input--> total bill amount and bill percent and it calculates the tip amount and the total bill.

Stack:
[x] Html
[x] CSS
[x] JavaScript

### PR DESCRIPTION
**Pull request by supzi-del**
Fixes #156

### CODE REVIEW

File: Tip Calculator/styles.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    font-weight:700;
+  }
+
+  #tip_amount{
+    margin-left:10px;
+  }
+
+  #bill_total{
+    margin-left:10px;
+  }
```
Comment (chicken-biryani) [inline Tip Calculator/styles.css:119]: Add a new line at the end of file. Use a CSS beautifier.

File: Tip Calculator/script.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    let tip = amount * (perc/100);
+
+
+    let total = tip + Number(amount);
+
+
+    document.getElementById('tip_amount').value = tip;
+    document.getElementById('bill_total').value = total;
+  }
```
Comment (chicken-biryani) [inline Tip Calculator/script.js:15]: Add a new line at the end of file.

Use a JS beautifier.

File: Tip Calculator/index.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          <input type="text" name="" id="bill_total" disabled>
+          <br>
+
+          <button onclick="tipcalc()" id="btn">Calculate</button>
+
+        </div><!--inputBox-->
+      </div><!--container-->
+  <script src="script.js"></script>
+</body>
+</html>
```
Comment (chicken-biryani) [inline Tip Calculator/index.html:38]: Add a new line at the end of the file. 
Use an HTML beautifier.

**Review by chicken-biryani — COMMENTED** [review]
1)Add project name in form of link and description in index.md
2) Contents of the README:

-Project overview.
-Use of the project.
-Stack of your project.
-How to set up / run your project.
-Screenshots of the pages/output.
(A short description under each screenshot... Describing the page a little bit)
This would help visitors/developers in general

3)Please use the following tools to increase the readability of your README / Scripts before creating a pull request.
It is a good practice.

Markdown - https://codebeautify.org/markdown-formatter
HTML - https://codebeautify.org/htmlviewer
CSS - https://codebeautify.org/css-beautify-minify
JS - https://codebeautify.org/jsviewer

Use this PR as a reference :
#93

**Review by Ayushparikh-code — APPROVED** [review]

**Review by Ayushparikh-code — CHANGES_REQUESTED** [review]
Add project name in form of link and description in index.md

**Comment by supzi-del** [comment]
Hey, what can I do about the conflicts?

**Review by harshita2216 — COMMENTED** [review]
I've resolved the conflicts.

**Comment by supzi-del** [comment]
Hey, can you please add the LGMSOC21 label so that this PR can be counted for the program?

**Pull request merged**